src/app/zap-templates/zcl/data-model/chip/oven-mode-cluster.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ModeTag" type="enum16">
+    <cluster code="0x0049"/>
+    <item value="0x4000" name="Bake"/>
+    <item value="0x4001" name="Convection"/>
+    <item value="0x4002" name="Grill"/>
+    <item value="0x4003" name="Roast"/>
+		<item value="0x4004" name="Clean"/>
+    <item value="0x4005" name="Convection Bake"/>
+    <item value="0x4006" name="Convection Roast"/>
+    <item value="0x4007" name="Warming"/>
+		<item value="0x4008" name="Proofing"/>
+  </enum>
+
+  <dataTypes>
```

Comment:
What is this and where is it coming from?  This is not something that happens in ZAP XML.